modules/config/src/main/scala/scala/cli/config/Key.scala

```diff
@@ -40,6 +40,8 @@ abstract class Key[T] {
 
   def description: String
 
+  def hidden: Boolean = false
```

Comment:
Add a short description/comment of what is `hidden` in the configuration key entries.

Reply:
Good catch, I added some scaladocs for new `Key` fields